src/css/sass.js

```diff
@@ -114,8 +127,67 @@ export default class SassCompiler extends CompilerBase {
     };
   }
 
+  buildImporterCallback (includePaths) {
+    const self = this;
+    return (function (request, done) {
+      let file
+      if (request.file) {
+        done();
+      } else {
+        // sass.js works in the '/sass/' directory
+        const cleanedRequestPath = request.resolved.replace(/^\/sass\//, '');
+        for (let includePath of includePaths) {
+          const filePath = path.resolve(includePath, cleanedRequestPath);
+          let variations = sass.getPathVariations(filePath);
+
+          file = variations
+            .map(self.fixWindowsPath.bind(self))
+            .reduce(self.importedFileReducer.bind(self), null);
+
+          if (file) {
+            const content = fs.readFileSync(file, { encoding: 'utf8' });
+            return sass.writeFile(file, content, () => {
+              done({ path: file })
+            });
+          }
+        }
+
+        if (!file) done();
+      }
+    });
+  }
+
+  importedFileReducer(found, path) {
+    // Find the first variation that actually exists
+    if (found) return found;
+
+    try {
+      const stat = fs.statSync(path);
+      if (!stat.isFile()) return null;
+      return path;
+    } catch(e) {
+      return null;
```

Comment:
Not sure about this - sass.js returns ~25 variations, each invalid one will fail here and trigger a debug log. In medium-sized projects with multiple sass files, this will generate a whole lot of unnecessary output. Correct me if I'm wrong :)